atlasdb-api/src/main/java/com/palantir/atlasdb/config/LockLeader.java

```diff
@@ -17,5 +17,9 @@
 
 public enum LockLeader {
     SOMEONE_ELSE_IS_THE_LOCK_LEADER,
-    I_AM_THE_LOCK_LEADER
+    I_AM_THE_LOCK_LEADER;
+
+    public static LockLeader fromBoolean(boolean isLockLeader) {
```

Comment:
It should be in the format `LockLeader.fromBoolean(x)` which does mean something though.